functions/venue.js

```diff
@@ -242,27 +261,44 @@ const createVenueData = (data, context) => {
   return venueData;
 };
 
-const createVenueData_v2 = (data, context) => ({
-  name: data.name,
-  config: {
-    landingPageConfig: {
-      coverImageUrl: data.bannerImageUrl,
-      subtitle: data.subtitle,
-      description: data.description,
+// @debt this should be de-duplicated + aligned with createVenueData to ensure they both cover all needed cases
+const createVenueData_v2 = (data, context) => {
+  const venueData_v2 = {
+    name: data.name,
+    config: {
+      landingPageConfig: {
+        coverImageUrl: data.bannerImageUrl,
+        subtitle: data.subtitle,
+        description: data.description,
+      },
     },
-  },
-  theme: {
-    primaryColor: data.primaryColor || DEFAULT_PRIMARY_COLOR,
-  },
-  host: {
-    icon: data.logoImageUrl,
-  },
-  owners: [context.auth.token.user_id],
-  showGrid: data.showGrid || false,
-  ...(data.showGrid && { columns: data.columns }),
-  template: data.template || VenueTemplate.partymap,
-  rooms: [],
-});
+    theme: {
+      primaryColor: data.primaryColor || DEFAULT_PRIMARY_COLOR,
+    },
+    host: {
+      icon: data.logoImageUrl,
+    },
+    owners: [context.auth.token.user_id],
+    showGrid: data.showGrid || false,
+    ...(data.showGrid && { columns: data.columns }),
+    template: data.template || VenueTemplate.partymap,
+    rooms: [],
+  };
+
+  // @debt find a way to share src/settings with backend functions, then use DEFAULT_SHOW_REACTIONS here
+  if (HAS_REACTIONS_TEMPLATES.includes(data.template)) {
+    venueData_v2.showReactions =
+      typeof data.showReactions === "boolean"
+        ? data.showReactions
+        : DEFAULT_SHOW_REACTIONS;
+    venueData_v2.showShoutouts =
```

Comment:
Whitespace between sibling blocks to increase readability:
```suggestion
        : DEFAULT_SHOW_REACTIONS;

    venueData_v2.showShoutouts =
```